### PR TITLE
DCT 247

### PR DESCRIPTION
**Pull request by EBEREGIT**
### `lastConsensusTime `

Added code sample

### CODE REVIEW

File: docs/src/rsh/compute/index.md [hunk 1]
```diff
@@ -1831,6 +1831,11 @@ lastConsensusTime()
 The @{defn("lastConsensusTime")} primitive returns the network time of the last publication of the DApp.
 This may not be available if there was no such previous publication, such as at the beginning of an application before the first publication.
 
+```reach
+load: /examples/realtime/index.rsh
+range: 17 - 17
```
Comment (TheChronicMonster) [inline docs/src/rsh/compute/index.md:1836]: This example does not add new information as compared to the syntactical reference of `lastConsensusTime`. Samples should provide an example of how the code may be used in a real DApp. 
The idea is that a Reach developer can copy and paste our sample and then modify per the needs of their program. 
In this example, we should show lines 12 - 18.

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

**Review by TheChronicMonster — DISMISSED** [review]

**Comment by jeapostrophe** [comment]
I like some introductory comment

File: docs/src/rsh/compute/index.md [hunk 2]
```diff
@@ -1836,6 +1836,8 @@ load: /examples/realtime/index.rsh
 range: 12 - 18
 ```
 
+Notice how `{!rsh} lastConsensusTime` is called on line 17 after the publication on line 12.
```
Comment (TheChronicMonster) [inline docs/src/rsh/compute/index.md:1839]: Please see [PR #993](https://github.com/reach-sh/reach-lang/pull/993/files/4ed2cf7c3f9dac6d9b75136f5bb62bb0b36285da)

For an example of an intuitive explanation of the sample. 

If you want to talk about the code samples please schedule a call with me.

**Review by jeapostrophe — APPROVED** [review]

**Comment by jeapostrophe** [comment]
This needs to be rebased manually I think

**Pull request closed without merging**